Makefile

```diff
@@ -297,6 +300,13 @@ darwin-update-examples:
 darwin-check-public-symbols:
 	node platform/darwin/scripts/check-public-symbols.js macOS iOS
 
+.PHONY: ios-clean-project
+ios-clean-project:
+	rm -fr $(IOS_PROJ_PATH)
```

Comment:
@alfwatt Can you say what the intention is behind these commands? Given that `make clean` already does this in a more obvious way, adding platform-specific commands seems unnecessary to me.

There certainly are some issues around where the iOS build directory/project lives and is managed (e.g., https://github.com/mapbox/mapbox-gl-native/pull/13718), but I think we’ll want to address those elsewhere.

Reply:
My understanding of the intended workflow was broken, though the js files were used to generate the xcodeproj not the other way around. Removing those targets.